Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM centos:7.5.1804
```

Comment:
We should pin to `centos:7` to pull in various security updates and to leverage layer caching a bit better (most images we use derive from `centos:7` already).